cairo/ethereum/cancun/vm/interpreter.cairo

```diff
@@ -157,16 +148,12 @@ func process_message{
     bitwise_ptr: BitwiseBuiltin*,
     keccak_ptr: KeccakBuiltin*,
     poseidon_ptr: PoseidonBuiltin*,
-}(message: Message, env: Environment) -> (Evm, EthereumException*) {
+}(message: Message, env: Environment) -> Evm {
     alloc_locals;
 
-    // Check if depth exceeds limit by checking if (depth - limit) is non-negative
-    let is_depth_exceeded = is_nn(message.value.depth.value - STACK_DEPTH_LIMIT);
-    if (is_depth_exceeded != FALSE) {
-        tempvar err = new EthereumException(StackDepthLimitError);
-        tempvar evm = Evm(cast(0, EvmStruct*));
-        return (evm, err);
-    }
+    // EELS checks whether the stack depth limit is reached here.
+    // However, this case is never triggered, because `generic_call` and `generic_create`
+    // check the stack depth limit before calling `process_message`.
```

Comment:
process_message can also be called by [process_message_call](https://github.com/ethereum/execution-specs/blob/d9febcb7332d62194968dcdb7775adf497c93626/src/ethereum/cancun/vm/interpreter.py#L90) right?

although normally, process_message_call is used only in tx initialization, which suggests it's unlikely the stack can be full

Reply:
exactly